samples/wifi/provisioning/implementation_details.rst

```diff
@@ -0,0 +1,268 @@
+Service Name               UUID
+Wi-Fi Provisioning Service 14387800-130c-49e7-b877-2881c89cb258
+========================== ====================================
+
+Service characteristics
+-----------------------
+
+The UUID value of characteristics are defined in table below.
+
+========================== ====================================
+Characteristic Name        UUID
+Information                14387801-130c-49e7-b877-2881c89cb258
+Operation Control Point    14387802-130c-49e7-b877-2881c89cb258
+Data Out                   14387803-130c-49e7-b877-2881c89cb258
+========================== ====================================
+
+The characteristic requirements of the Wi-Fi Provisioning Service are shown in table below.
+
++-----------------+-------------+-------------+-------------+-------------+
+| Characteristic  | Requirement | Mandatory   | Optional    | Security    |
+| Name            |             | Properties  | Properties  | Permissions |
++=================+=============+=============+=============+=============+
+| Information     | M           | Read        |             | No Security |
+|                 |             |             |             | required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | M           | Indicate,   |             | Encryption  |
+| Control         |             | Write       |             | Required    |
+| Point           |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Operation       | M           | Read, Write |             | Encryption  |
+| Control         |             |             |             | Required    |
+| Point           |             |             |             |             |
+| - Client        |             |             |             |             |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Notify      |             | Encryption  |
+|                 |             |             |             | Required    |
++-----------------+-------------+-------------+-------------+-------------+
+| Data Out        | M           | Read, Write |             | Encryption  |
+| - Client        |             |             |             | Required    |
+| Characteristic  |             |             |             |             |
+| Configuration   |             |             |             |             |
+| descriptor      |             |             |             |             |
++-----------------+-------------+-------------+-------------+-------------+
+
+Information
+-----------
+
+This characteristic is used to get Info message.
```

Comment:
'Service' in the sample name gets confusing. 

Fix capitalization at the start of a new sentence.

I don't think we need to specify it's a protocol buffer message because it's the only kind of message we've talked about.